src/edit/tools/DistortableImage.Guides.js

```diff
@@ -0,0 +1,8 @@
+L.DistortableImage = L.DistortableImage || {};
+
+// keymapper
+var dom_string = "<b><center><h3>Keymappings</h3><center><hr/></center></center><ul><li>L: Lock overlay</li><li>O: Outline overlay</li><li>R: Rotate overlay</li><li>RR: Distort overlay</li><li>T: Transparent overlay&nbsp;&nbsp;&nbsp;&nbsp;</li><li>DEL: Remove overlay</li></ul></b>";
```

Comment:
we can think about a better way to structure this in the future by defining variables and sticking them in to the string, but for now lets at least increase readability by doing something like this: 
```suggestion
var dom_string = 
"<b><center><h3>Keymappings</h3><center><hr/></center></center><ul>" + 
"<li>L: Lock overlay</li>" + "<li>O: Outline overlay</li>" + "<li>R: Rotate overlay</li>" +
"<li>RR: Distort overlay</li>" + "<li>T: Transparent overlay&nbsp;&nbsp;&nbsp;&nbsp;</li>" + 
"<li>DEL: Remove overlay</li></ul></b>";
```